From 3fe0fb5a21861d728da229a0112319012096d840 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Robert-Andr=C3=A9=20Mauchin?= <zebob.m@gmail.com>
Date: Fri, 5 Oct 2018 17:40:44 +0200
Subject: [PATCH] Fix "unknown field MinimumLevelToLog" error

MinimumLevelToLog has been removed in azure-pipeline-go v0.1.8.
Now replaced by ShouldLog.
---
 2016-05-31/azblob/zt_examples_test.go | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/2016-05-31/azblob/zt_examples_test.go b/2016-05-31/azblob/zt_examples_test.go
index 1ab3051..5be7742 100644
--- a/2016-05-31/azblob/zt_examples_test.go
+++ b/2016-05-31/azblob/zt_examples_test.go
@@ -143,7 +143,9 @@ func ExampleNewPipeline() {
 				// This method is not called for filtered-out severities.
 				logger.Output(2, m) // This example uses Go's standard logger
 			},
-			MinimumLevelToLog: func() pipeline.LogLevel { return pipeline.LogInfo }, // Log all events from informational to more severe
+			ShouldLog: func(level pipeline.LogLevel) bool {
+				return level <= pipeline.LogInfo // Log all events from informational to more severe
+			},
 		},
 	}
 
-- 
2.19.0

